src/components/TextInput/styled.tsx

```diff
@@ -13,26 +14,11 @@ export const Container = styled.div`
   width: 100%;
 `;
 
-export const Input = styled.input`
-  flex: 1;
-  display: flex;
-  margin: 0;
-  padding: 0;
-  outline: none;
-  border: none;
-  text-decoration: none;
-  background: transparent;
-  color: inherit;
-  height: ${({ theme }) => em(theme.honeycomb.size.huge, theme.honeycomb.size.reduced)};
-  text-indent: ${({ theme }) => em(theme.honeycomb.size.small, theme.honeycomb.size.reduced)};
-  padding-right: ${({ theme }) => em(theme.honeycomb.size.small, theme.honeycomb.size.reduced)};
-  font-size: ${({ theme }) => em(theme.honeycomb.size.reduced)};
-
-  ${({ theme }) => transitions(['color'], `${theme.honeycomb.duration.normal} ease-in-out`)};
-
-  ::placeholder {
-    color: ${({ theme }) => theme.honeycomb.color.text.masked};
-  }
+export const Description = styled.span`
+  display: block;
+  color: ${({ theme }) => theme.honeycomb.color.text.masked};
+  font-size: ${({ theme }) => em(theme.honeycomb.size.small)};
+  margin-top: ${({ theme }) => em(4, theme.honeycomb.size.small)};
```

Comment:
```suggestion
  margin-top: ${({ theme }) => em(theme.honeycomb.size.micro, theme.honeycomb.size.small)};
```

^ We have a few new sizes in the theme now.